packages/easy-coding-standard/packages/sniff-runner/src/ValueObject/File.php

```diff
@@ -191,9 +190,7 @@ public function addError($error, $stackPtr, $code, $data = [], $severity = 0, $f
     }
 
     /**
-     * Allow only specific classes
-     *
-     * {@inheritdoc}
+     * Allow only specific classes  {@inheritdoc}
```

Comment:
`{@Inheritdoc}` should be on different line

Reply:
Another case where the check isn't good enough maybe; line starting with `'{@'` should also not be touched.